src/test/scala/no/ndla/searchapi/service/search/MultiDraftSearchServiceTest.scala

```diff
@@ -173,8 +173,8 @@ class MultiDraftSearchServiceTest extends IntegrationSuite with TestEnvironment
         multiDraftSearchSettings.copy(query = Some("bil"), sort = Sort.ByRelevanceDesc))
     val hits = results.results
     results.totalCount should be(3)
-    hits.head.id should be(5)
-    hits(1).id should be(1)
+    hits.head.id should be(1)
```

Comment:
Vet vi hva som gjør at scoringen har endret seg her? Er det ok?

Reply:
Veit egentlig ikkje kvifor dette er endra. Det kan ha sammenheng med at eg ny tokenizer for nynorsk. Kan teste å revertere det for å sjekke om det var det.

Reply:
Hadde ikkje noko å sei.

Reply:
Når eg ser på testdata har artikkel 1 tre innslag av ordet bil mens artikkel 5 har berre to. Så scoringa ser korrekt ut.